### PR TITLE
add gen-bs.0.1.0

### PR DESCRIPTION
**Pull request by 0zat**

### CODE REVIEW

**Comment by 0zat** [comment]
excuse me, how can I troubleshoot for "depfail"?

**Comment by yallop** [comment]
@0zat: if you follow the links to the CI logs under "Some checks were not successful" you'll end up [here](https://ci.ocaml.io/ocaml/opam-repository/pr/10055?test=1%20V1.2%20Build) and then [here](https://ci.ocaml.io/log/saved/docker-build-11fe994a53629044ac62853c2b0e7459/261ab2c10a94e83acaed08863163909b4ba06718), where you'll see this:


```
# ocamlfind ocamldep -package webidl -package batteries -package yojson -modules src/main/gen_bs.ml > src/main/gen_bs.ml.depends
# + ocamlfind ocamldep -package webidl -package batteries -package yojson -modules src/main/gen_bs.ml > src/main/gen_bs.ml.depends
# ocamlfind: Package `yojson' not found
# Command exited with code 2.
# setup.ml exec:ocamlbuild -use-ocamlfind gen_bs.byte

```

which suggests that you have a missing `yojson` entry in the `depends` section of your `opam` file.

File: packages/gen-bs/gen-bs.0.1.0/opam [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+license: "MIT"
+dev-repo: "git+https://github.com/0zat/gen-bs.git"
+build: ["ocaml" "setup.ml" "build"]
+install: ["ocaml" "setup.ml" "install"]
+remove: ["ocaml" "setup.ml" "remove"]
+depends: [
+  "ocamlfind" {build & >= "1.7.1"}
+  "ocamlbuild" {build & >= "0.9.3"}
+  "batteries" {>= "2.6.0"}
+  "webidl" {>= "1.3"}
```
Comment (avsm) [inline packages/gen-bs/gen-bs.0.1.0/opam]: adding yojson here should fix the problem

**Comment by 0zat** [comment]
@yallop , @avsm 
Thank you for the advice.
I have added yojson depend.

**Pull request merged**